conda/recipes/librmm/recipe.yaml

```diff
@@ -0,0 +1,105 @@
+        - AWS_ACCESS_KEY_ID
+        - AWS_SECRET_ACCESS_KEY
+        - AWS_SESSION_TOKEN
+
+      env:
+        CMAKE_C_COMPILER_LAUNCHER: ${{ env.get("CMAKE_C_COMPILER_LAUNCHER", default="sccache") }}
+        CMAKE_CUDA_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CUDA_COMPILER_LAUNCHER", default="sccache") }}
+        CMAKE_CXX_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CXX_COMPILER_LAUNCHER", default="sccache") }}
+        CMAKE_GENERATOR: ${{ env.get("CMAKE_GENERATOR", default="Ninja") }}
+        PARALLEL_LEVEL: ${{ env.get("PARALLEL_LEVEL", default="4") }}
+        SCCACHE_BUCKET: ${{ env.get("SCCACHE_BUCKET", default="") }}
+        SCCACHE_IDLE_TIMEOUT: ${{ env.get("SCCACHE_IDLE_TIMEOUT", default="") }}
+        SCCACHE_REGION: ${{ env.get("SCCACHE_REGION", default="") }}
+        SCCACHE_S3_USE_SSL: ${{ env.get("SCCACHE_S3_USE_SSL", default="") }}
+        SCCACHE_S3_NO_CREDENTIALS: ${{ env.get("SCCACHE_S3_NO_CREDENTIALS", default="") }}
+        SCCACHE_S3_KEY_PREFIX: librmm-${{ env.get("RAPIDS_CONDA_ARCH", default="") }}
+
+outputs:
+  - package:
+      name: librmm
+      version: ${{ version }}
+    build:
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+      script: install_librmm.sh
+    requirements:
+      host:
+        - cmake ${{ cmake_version }}
+        - cuda-version =${{ cuda_version }}
+      run:
+        - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+        - if: cuda_major == "11"
+          then: cudatoolkit
+        - rapids-logger =0.1
+      run_exports:
+        - ${{ pin_subpackage("librmm", upper_bound="x.x") }}
+      ignore_run_exports:
+        from_package:
+          - ${{ compiler("cuda") }}
+    tests:
+      - script:
+          - "test -d \"${PREFIX}/include/rmm\""
+    about:
+      homepage: ${{ load_from_file("python/librmm/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/librmm/pyproject.toml").project.license.text | replace(" ", "-") }}
+      summary: ${{ load_from_file("python/librmm/pyproject.toml").project.description }}
+  - package:
+      name: librmm-tests
+      version: ${{ version }}
+    build:
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+      script: install_librmm_tests.sh
+    requirements:
+      host:
+        - cmake ${{ cmake_version }}
+        - cuda-version =${{ cuda_version }}
+        - if: cuda_major == "11"
+          then: cudatoolkit
+          else: cuda-cudart-dev
+      run:
+        - if: cuda_major == "11"
+          then: cudatoolkit
+          else: cuda-cudart
+        - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+        - ${{ pin_subpackage("librmm", exact=True) }}
+      ignore_run_exports:
+        from_package:
+          - ${{ compiler("cuda") }}
+          - if: cuda_major == "11"
+            then: cuda-cudart-dev
+    about:
+      homepage: ${{ load_from_file("python/librmm/pyproject.toml").project.urls.Homepage }}
```

Comment:
Can we `load_from_file` once into something in the context and then access that data? I don't know if rattler is smart enough to cache the file on its own.

Reply:
Doesn't seem to work.  In all of their documentation examples they repeatedly call `load_from_file` on the same file

Reply:
Weird. I opened https://github.com/prefix-dev/rattler-build/issues/1423. This seems like a bug in the Jinja parsing.

Reply:
Nothing we can do here for now.